clusterloader2/pkg/measurement/common/profile.go

```diff
@@ -221,6 +219,26 @@ func (p *profileMeasurement) shouldExposeApiServerDebugEndpoint() bool {
 	return p.config.componentName == "kube-apiserver"
 }
 
+func (p *profileMeasurement) getProfileCommand() (string, error) {
+	profilePort, err := getPortForComponent(p.config.componentName)
+	if err != nil {
+		return "", fmt.Errorf("get profile command failed finding component port: %v", err)
```

Comment:
NIT: Could you use go-errors for all added / updated lines? Here is an example https://github.com/kubernetes/perf-tests/blob/75dfad9672dfabe11b292c246f8f99c79db84655/clusterloader2/pkg/config/template_provider.go#L240